packages/electron-updater/src/types.ts

```diff
@@ -0,0 +1,81 @@
+import { CancellationToken, PackageFileInfo, ProgressInfo, UpdateFileInfo, UpdateInfo } from "builder-util-runtime"
```

Comment:
Please export `ProgressInfo` and `UpdateInfo` in the `electron-updater` package

My app depends on `UpdateInfo` and `ProgressInfo` types, and I noticed that this change removed the exports of these two types :(

Reply:
Oh no! Sorry about that, I must have missed the re-export. Will get that into the next release asap